rfc/src/intro.md

```diff
@@ -44,22 +46,24 @@ This is the overall workflow for the RFC process:
    3. If the RFC is not approved, close the PR without merging.
 3. Feature implementation.
    1. Start implementing the new feature in your fork.
-   2. It is OK to implement it incrementally over multiple PRs. Just ensure that every pull request has a testable
-      end-to-end flow and that it is properly tested.
-   3. In the implementation stage, the feature should only be accessible if the user explicitly passes
-      `--enable-unstable` as an argument to Kani.
-   4. Document how to use the feature.
-   5. Keep the RFC up-to-date with the decisions you make during implementation.
+   2. Create a new revision of the RFC to add details about the "Software Design".
+   3. It is OK to implement the feature incrementally over multiple PRs.
+      Just ensure that every pull request has a testable end-to-end flow and that it is properly tested.
```

Comment:
... and that you update the RFC accordingly if there are any changes in the proposed design.

Reply:
oh, you already mention that on point 6